apps/prairielearn/src/tests/access.test.js

```diff
@@ -4,6 +4,7 @@ var request = require('request');
 var cheerio = require('cheerio');
 
 const { config } = require('../lib/config');
+const { insertEnrollment } = require('../models/enrollment');
```

Comment:
Since I was already introducing a new `insertEnrollment` function in this PR, I opted to replace the other bespoke implementations of that function with the single new one.

Reply:
❤️ these continual cleanups!